rbac/management/audit_log/model.py

```diff
@@ -138,3 +153,17 @@ def log_edit(self, request, resource, object):
         self.action = AuditLog.EDIT
         self.tenant_id = self.get_tenant_id(request)
         super(AuditLog, self).save()
+
+    def log_add(self, request, resource, object, type_dict, user_type):
```

Comment:
maybe better name ?  it is not visible difference between `log_add` and `log_create`. 
something like "log_role_or_user_added_to_group" or `log_group_membership` or `log_group_assigment`.

Reply:
Oh! There's another task to add permissions to a role, so I was thinking of having this ` log_add ` function do that as well! I just have it for now grouped this way to split the prs.